pom.xml

```diff
@@ -73,13 +73,13 @@
         <dependency>
             <groupId>com.aws.greengrass</groupId>
             <artifactId>nucleus</artifactId>
-            <version>2.6.0-SNAPSHOT</version>
+            <version>2.7.0-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.aws.greengrass</groupId>
             <artifactId>nucleus</artifactId>
-            <version>2.6.0-SNAPSHOT</version>
+            <version>2.7.0-SNAPSHOT</version>
```

Comment:
Depends on a new method on the security service of the Nucleus [see here](https://github.com/aws-greengrass/aws-greengrass-nucleus/commit/c4cfe2bf8dace146556d2ddabd4dd90b474004f7#diff-d808e9ba1be2c30f79fe423fcc84d462f988c5696bd699894ea720d993c333faR179). Only available after snapshot 2.7.0.